modules/platforms/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+#
```

Comment:
Two general questions:
- Is this a common practice to use `*.txt` files for CMake? I never used it, but it looks weird to me.
- `platforms` modules should probably contain code related to thin clients. I'd rather have a different module for this code to be honest.

Reply:
* It's weird for me too but this is how CMake works.
* Probably you are right. I'd appreciate any advise on module placement. If client developers would re-use this module, what is the best location for them?

Reply:
We should ask them probably, I don't know the answer

Reply:
Well, maybe it's Ok to store all C++ code in a single place for easier re-use. We just need to make sure that we have decent modularity here and that we can easily configure compilation so that we won't need to wait to compile all the code when we only work on our specific client or store code.